07/index.html

```diff
@@ -0,0 +1,136 @@
+        </fieldset>
+        <fieldset>
+          <legend>Doble Propósito</legend>
+          <label for="V-Strom800">V-Strom 800DE</label>
+          <input type="radio" id="V-Strom800" name="Dp" value="v-strom-800">
+          <h4>254,990 mxn</h4>
+          <img src="./assets/example/v-strom-800de.png" width="200">
+          <br>
+          <label for="dr650">DR 650</label>
+          <input type="radio" id="dr650" name="Dp" value="dr650">
+          <h4>169,990 mxn</h4>
+          <img src="./assets/example/dr650.png" width="200">
+          <br>
+          <label for="250sx">V-Strom 250SX</label>
+          <input type="radio" id="250sx" name="Dp" value="250sx">
+          <h4>99,990 mxn</h4>
+          <img src="./assets/example/v-strom-250-sx.png" width="200">
+        </fieldset>
+        <fieldset>
+          <legend>Scooter</legend>
+          <label for="Avenis">Avenis 125</label>
+          <input type="radio" id="Avenis" name="scooter" value="avenis">
+          <h4>49,990 mxn</h4>
+          <img src="./assets/example/avenis-125.png" width="200">
+          <br>
+          <label for="burgman">Brugman street 125</label>
+          <input type="radio" id="burgman" name="scooter" value="burgman">
+          <h4>44,990 mxn</h4>
+          <img src="./assets/example/burgman-street-125.png" width="200">
+        </fieldset>
+      </fieldset>
+    </section>
+    <article>
+      <fieldset>
+        <legend id="servicio">Servicios</legend>
+        <label for="Snombre">Nombre y apellido</label>
+        <input type="text" id="Snombre">
+        <br>
+        <label for="E-mail">E-mail</label>
+        <input type="email" id="E-mail">
+        <br>
+        <label for="telefono">Numero telefonico</label>
+        <input type="tel" id="telefono">
+        <br>
+        <br>
+        <select>Modelos
+          <option selected disabled>Seleccione un modelo</option>
+          <optgroup label="Cuidad">
+            <option value="Gsx-8s">Gsx-8s</option>
+            <option value="Intruder">Intruder</option>
+            <option value="Gixxer 150">Gixxer 150</option>
+          </optgroup>
+          <optgroup label="Doble Propósito">
+            <option value="V-Strom800">V Strom 800DE</option>
+            <option value="DR 650">DR 650</option>
+            <option value="V Strom 250 SX">V Strom 250 SX</option>
+          </optgroup>
+          <optgroup label="Scooter">
+            <option value="Avenis">Avenis</option>
+            <option value="Brugman">Brugman</option>
+          </optgroup>
+        </select>
+        <br>
+        <br>
+        <select>Kilometros
+          <option selected disabled>Seleccione un kilometraje</option>
+          <option>1,000</option>
+          <option>4,000</option>
+          <option>8,000</option>
+          <option>12,000</option>
+          <option>16,000</option>
+          <option>20,000</option>
+          <option>24,000</option>
```

Comment:
There is a missing attr for theses options
---
Hay un atributo faltante para estas opciones